src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -108,7 +108,7 @@ public function normalize($object, $format = null, array $context = array())
      *
      * @return string[]
      */
-    protected function getAttributes($object, $format = null, array $context)
+    protected function getAttributes($object, $format, array $context)
```

Comment:
What is the reason for this change? This is breaking compatibility with classes that extend the `AbstractObjectNormalizer`.